lib/std/build.zig

```diff
@@ -1276,40 +1214,69 @@ fn isLibCLibrary(name: []const u8) bool {
     return false;
 }
 
+/// A file that is generated by a build step.
+/// This struct is an interface that is meant to be used with `@fieldParentPtr` to implement the actual path logic.
+pub const GeneratedFile = struct {
+    /// The step that generates the file
+    step: *Step,
+
+    /// The path to the generated file. Must be either absolute or relative to the build root.
+    /// This value must be set in the `fn make()` of the `step` and must not be `null` afterwards.
+    path: ?[]const u8 = null,
+
+    pub fn getPath(self: *const GeneratedFile) []const u8 {
+        return self.path orelse @panic("getPath() was called on a GeneratedFile that wasn't build yet. Is there a missing Step dependency?");
```

Comment:
This error message should give info about the step that failed, since you have it available.

Reply:
I added the name of the step into the panic message